engines/wintermute/base/base_keyboard_state.cpp

```diff
@@ -342,8 +375,10 @@ uint32 BaseKeyboardState::keyCodeToVKey(Common::Event *event) {
 	case Common::KEYCODE_F12:
 		return kVkF12;
 	default:
-		warning("Key not handled: %d '%c'", event->kbd.keycode, event->kbd.keycode);
-		return event->kbd.keycode;
+		if (!((event->kbd.flags & Common::KBD_NON_STICKY) != 0)) {
```

Comment:
Doesn't ! and != 0 boil down to the much simpler statement: if ((event->kbd.flags & Common::KBD_NON_STICKY) == 0)